### PR TITLE
Enable missing_whitespace_between_adjacent_strings

### PR DESCRIPTION
**Pull request by goderbauer**
This one is catching some real bugs, but also requires some ignores in other places...

https://github.com/flutter/flutter/issues/75663

### CODE REVIEW

File: dev/benchmarks/macrobenchmarks/lib/src/fullscreenTextField.dart [hunk 1]
```diff
@@ -32,19 +32,19 @@ const String textLotsOfText = 'Lorem ipsum dolor sit amet, consectetur '
...
+  '️😣 😖 😫 😩 🥺 😢 😭 😤 😠 😡 🤬 🤯 😳 🥵 🥶 😱 😨 😰 😥 😓 '
   '🤗 🤔 🤭 🤫 🤥 😶 😐 😑 😬 🙄 😯 😦 😧 😮 😲 🥱 😴 🤤 😪 😵 '
   '🤐 🥴 🤢 🤮 🤧 😷 🤒 🤕 🤑 🤠 😈 👿 👹 👺 🤡 💩 👻 💀 ☠️ 👽 '
-  '👾 🤖 🎃 😺 😸 😹 😻 😼 😽 🙀 😿 😾'
-  '👋 🤚 🖐 ✋ 🖖 👌 🤌 🤏 ✌️ 🤞 🤟 🤘 🤙 👈 👉 👆 🖕 👇 ☝️ 👍 👎'
-  ' ✊ 👊 🤛 🤜 👏 🙌 👐 🤲 🤝 🙏 ✍️ 💅 🤳 💪 🦾 🦵 🦿 🦶 👣 👂 '
+  '👾 🤖 🎃 😺 😸 😹 😻 😼 😽 🙀 😿 😾 '
+  '👋 🤚 🖐 ✋ 🖖 👌 🤌 🤏 ✌️ 🤞 🤟 🤘 🤙 👈 👉 👆 🖕 👇 ☝️ 👍 👎 '
+  '✊ 👊 🤛 🤜 👏 🙌 👐 🤲 🤝 🙏 ✍️ 💅 🤳 💪 🦾 🦵 🦿 🦶 👣 👂 '
   '🦻 👃 🫀 🫁 🧠 🦷 🦴 👀 👁 👅 👄 💋 🩸';
```
Comment (Hixie) [inline dev/benchmarks/macrobenchmarks/lib/src/fullscreenTextField.dart:48]: will this affect the benchmark numbers?

File: dev/bots/test/analyze-test-input/root/packages/foo/deprecation.dart [hunk 2]
```diff
@@ -9,7 +9,7 @@
 void test1() { }
 
 @Deprecated(
-  'Missing space ->.'
+  'Missing space ->.' //ignore: missing_whitespace_between_adjacent_strings
```
Comment (Hixie) [inline dev/bots/test/analyze-test-input/root/packages/foo/deprecation.dart:12]: this ignore may interfere with the code that is checking this. Might be a good place to use a file-wide ignore.

**Review by Hixie — APPROVED** [review]
LGTM

File: dev/benchmarks/macrobenchmarks/lib/src/fullscreenTextField.dart (hunk 1 above)
Comment (goderbauer) [reply dev/benchmarks/macrobenchmarks/lib/src/fullscreenTextField.dart:48]: I wouldn't expect it to, will keep an eye on the benchmark, though.

File: dev/bots/test/analyze-test-input/root/packages/foo/deprecation.dart (hunk 2 above)
Comment (goderbauer) [reply dev/bots/test/analyze-test-input/root/packages/foo/deprecation.dart:12]: Will change to file-wide ignore if any checks fail...

**Pull request merged**